Makefile

```diff
@@ -12,6 +19,24 @@ format:
 	shfmt -w tests/test-svmkit
 	shfmt -w bin/check-env
 
+docker-run: docker-image
+	docker run --platform=linux/$(ARCH) $(TAG) start --tags "queue=$(USER)"  --token "$(BUILDKITE_API_TOKEN)"
+
+docker-build:
+	docker build --platform=linux/$(PLATFORM) -t $(TAG) -f $(DOCKERFILE) $(BUILD_CONTEXT)
+
+# Buildx multi-platform build (does NOT push by default)
+docker-buildx:
+	docker buildx build \
+		--platform=$(PLATFORMS) \
+		-t $(TAG) \
+		-f $(DOCKERFILE) \
+		$(BUILD_CONTEXT)
+
+.Dockerfile: Dockerfile
```

Comment:
For this target, when is this getting built?  And if it's getting built, when would it get cleaned up?